app/views/posts/_post_comment.html.erb

```diff
@@ -0,0 +1,3 @@
+<div>
+  <%= comment.author.name %> : <%= comment.text %></p>
+</div>
```

Comment:
- [ ] Let's make use of `recent_comments` callback function in our Post model to render the 5 most comments for a post. I created comments for a post but I did not see them.